test/parallel/test-buffer-copy.js

```diff
@@ -96,9 +96,14 @@ bb.fill('hello crazy world');
 b.copy(c, 0, 100, 10);
 
 // copy throws at negative sourceStart
-assert.throws(function() {
-  Buffer.allocUnsafe(5).copy(Buffer.allocUnsafe(5), 0, -1);
-}, RangeError);
+common.expectsError(
+  () => Buffer.allocUnsafe(5).copy(Buffer.allocUnsafe(5), 0, -1),
+  {
```

Comment:
The contents of the object can be stored in a const and used on lines 101 and 122:
```
{
    code: 'ERR_OUT_OF_RANGE',
    type: RangeError,
    message: 'Index out of range'
}
```

Reply:
Thank you. Done